versions/3.0.md

```diff
@@ -107,18 +107,18 @@ For example, if a field has an array value, the JSON array representation will b
 }
 ```
 
-While the API is described using JSON, it does not impose a JSON input/output to the API itself.
+While the API is described using JSON, input to and output from the API is not required to be JSON.
```

Comment:
This sentence always bothered me because I think it is out of place. I suggest moving it to the end of the section, and perhaps making it stand out as a side note:

**Note**: While APIs are *described* with JSON or YAML Open API documents, the APIs' request and response bodies and other content are not required to be JSON or YAML.